src/boot/efi/chid.h

```diff
@@ -0,0 +1,16 @@
+/* SPDX-License-Identifier: BSD-3-Clause */
+#pragma once
+
+#include "efi.h"
+
+#include "chid-fundamental.h"
+
+typedef struct Device {
+        char name[128];        /* Arbitrary size, shoule be big enough to fit any device name */
```

Comment:
what is this used for? purely decorative purposes?

Reply:
Yeah, basically just for logging.